examples/pom.xml

```diff
@@ -18,12 +18,13 @@
   <properties>
     <protobuf.output.directory>${project.build.directory}/generated-sources</protobuf.output.directory>
     <protobuf.input.directory>${project.basedir}/proto</protobuf.input.directory>
-    <java.version>11</java.version>
+    <java.version>17</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
```

Comment:
can the target still be 8/11? why does that need to change?

Reply:
As we discussed, we will use jdk17 and springboot3 for examples and sdk-tests subprojects.